assets/stylesheets/components/_jumbotrons.scss

```diff
@@ -18,6 +18,19 @@
     }
   }
 
+  &.topo-overlay {
+    &:before {
+      background-image: url('http://crds-media.imgix.net/1KLwsN6ilL7RO0xk6JzOsX/840e8b99340ff36bff67036909c4f856/texture-topo-dark-100_2x.png');
```

Comment:
This asset should be loaded agnostic of protocol (ie. use `//...` instead of `http://...`)